perfact/zodbsync/tests/test_sync.py

```diff
@@ -1215,3 +1215,73 @@ def test_no_meta_file(self):
 
         with pytest.raises(AssertionError):
             self.run('playback', '/foo')
+
+    def test_force_default_owner(self):
+        """
+        Check if the default owner can be forced via config
+        """
+
+        self.runner.sync.force_default_owner = True
+
+        # first test: owner from meta file pushed to app
+        folder = os.path.join(self.repo.path, '__root__', 'newfolder')
+        os.mkdir(folder)
+
+        with open(os.path.join(folder, '__meta__'), 'w') as f:
+            f.write(zodbsync.mod_format({
+                "title": "",
+                "type": "Folder",
+                "owner": "Somebody",  # i used to know
+            }))
+
+        self.run('playback', '/newfolder')
+
+        expected_owner = (['acl_users'], self.runner.sync.default_owner)
+
+        assert self.app.newfolder._owner == expected_owner
+
+        # second test: owner from zope read to meta file
+        with self.runner.sync.tm:
+            self.app.manage_addProduct['OFSP'].manage_addFolder(id='another')
+
+        self.app.another._owner = 'Somebody'
```

Comment:
This is not a valid owner (it should be a tuple), but I guess this does not really matter here.